src/Components/Projects.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import ProjectItem from './ProjectItem';
+
+class Projects extends Component {
+  render() {
+    let projectItems;
+    if(this.props.projects){
+        projectItems = this.props.projects.map(project => {
+          //console.log(project);
+          return (
+            <ProjectItem key={project.title} project={project} />
+          );
+        });
+    }
```

Comment:
We can also do this in `return` block below.

Reply:
Good to know